### ISSUE TITLE
datasource doesnt seem to do anything

### ISSUE THREAD

**Issue by Rolf-Herbert** [rykdesjardins/js-calendar#2]
Hi there,

I have a couple of issues.
1 - If I am planning to fetch the matrix from an ajax call are there any other alternative matrix layouts, like a json structure. Also do I have to use date objects because that will mean evaluating the response.

2 - when I enter datasource:'mypage.htm' in the options nothing happens and there is no 'hit' on that page. <-- **worked that out you have to call setMatrix() in the chain**

I guess an example of using an external source for the data matrix and the format of the transmitted data would be ace.
Thanks

**Comment by murdoch** [comment]
Agreed, it would be cool to see a demo page where the events are pulled, via ajax, as json. The project looks promising though. Nice work.

**Comment by Rolf-Herbert** [comment]
I love the speed of the calendar but the data format for the matrix is odd and would be awkward to replicate from a database query, that DATE object as well is a problem if getting from an external source.
